features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/resource-access-control-v2.xml.j2

```diff
@@ -318,6 +318,9 @@
             <Permissions>/permission/admin/manage/identity/rolemgt/view</Permissions>
             <Scopes>internal_org_role_mgt_view</Scopes>
         </Resource>
+        <Resource context="(.*)/o/scim2/v2/Roles(.*)" secured="true" http-method="POST">
+            <Permissions>/permission/admin/manage/identity/rolemgt/create</Permissions>
```

Comment:
permission string are not needed in this file. only scopes are needed. Let's avoid adding them for new configs unless there is a reason
Let's confirm and remove the unwanted permissions in old role mgt API resources aswell.